### DISCUSSION TITLE
【宠物挂件】 怎么改点击宠物跳转的地址

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#2146] [Q&A] [answered]
点击博客的挂件宠物，默认会切换主题，如果希望点击后跳转到别的页面怎么办？

**Comment by tangly1024** [comment, marked as answer]
警告，这是未被配置支持的功能，通过修改源代码实现，可能会出现未知的编译错误。

## 修改/components/Live2D.js 文件的两行代码

### 修改位置1

第33行：

https://github.com/tangly1024/NotionNext/blob/a33bf9f4d4011c32744a098fbd3c8396da3194f9/components/Live2D.js#L33-L37


修改为
```
 function handleClick() { 
    window.location.href = "https://blog.tangly1024.com";
 } 
```

### 修改位置2

第12行

https://github.com/tangly1024/NotionNext/blob/a33bf9f4d4011c32744a098fbd3c8396da3194f9/components/Live2D.js#L12

修改为：
```
  const { theme } = useGlobal()
```

**Reply by darcyli** [reply]
![image](https://github.com/tangly1024/NotionNext/assets/59253940/e34e189d-0a7a-4712-a91a-bec8cfc5534a)
尝试了一下，报错了😔
不知道后续版本会考虑修改掉Live2D这个默认设置吗？HEXO主题点击后跳到了landing主题，显示内容和个人博客的毫无干系了
